src/examples/basic.py

```diff
@@ -9,14 +9,15 @@ class MyEnvClass(NetworkBase):
     """
 
     def create_hook(self):
-        # Add a bastion host as a child stack in the environment
-        self.add_child_template(Bastion())
+        pass
+        # # Add a bastion host as a child stack in the environment
```

Comment:
@gimballock what do you say to this ^ comment?  Does the basic example in master work as it is? or do we need to modify it to make it work?  Or else, why did you add that commit to this PR?